src/include/hip/rpp_hip_common.hpp

```diff
@@ -1,10 +1,12 @@
 #ifndef RPP_HIP_COMMON_H
 #define RPP_HIP_COMMON_H
 
+#include "hip/rpp/handle.hpp"
 #include <hip/hip_runtime_api.h>
 #include <hip/hip_runtime.h>
 #include <hip/hip_ext.h>
-#include <hip/hip_fp16.h>
+#include <hip/hcc_detail/hip_vector_types.h>
```

Comment:
please don't include files from the hcc_detail folder from the hip. including from the hip/hip_fp16 header should give you access to all the HIP's fp16 functionalities.

Reply:
Reverted back the headers without hcc_detail